code-challenges.test.js

```diff
@@ -15,42 +15,137 @@
 
 // a) Create a test with expect statements using the variables provided.
 
-const secretCodeWord1 = "Lackadaisical"
-// Expected output: "L4ck4d41s1c4l"
-const secretCodeWord2 = "Gobbledygook"
-// Expected output: "G0bbl3dyg00k"
-const secretCodeWord3 = "Eccentric"
-// Expected output: "3cc3ntr1c"
-
+describe("codedMessage", () => {
+    it("takes in a string and returns a coded message. The coded message converts 'a' to 4, 'e' to 3, 'i' to 1, and 'o' to 0.", () => {
+        const secretCodeWord1 = "Lackadaisical"
+        // Expected output: "L4ck4d41s1c4l"
+        const secretCodeWord2 = "Gobbledygook"
+        // Expected output: "G0bbl3dyg00k"
+        const secretCodeWord3 = "Eccentric"
+        // Expected output: "3cc3ntr1c"
+      expect(codedMessage(secretCodeWord1)).toEqual("L4ck4d41s1c4l")
+      expect(codedMessage(secretCodeWord2)).toEqual("G0bbl3dyg00k")
+      expect(codedMessage(secretCodeWord3)).toEqual("3cc3ntr1c")
+    })
+  })
+
+//   ● codedMessage › takes in a string and returns a coded message. The coded message converts 'a' to 4, 'e' to 3, 'i' to 1, and 'o' to 0.
+
+//     ReferenceError: codedMessage is not defined
 
 // b) Create the function that makes the test pass.
 
+//Create a function - codedMessage
+//Parameter - string 
+//Go through string recognizing vowels 
+//Use the replace method to replace lowercase and uppercase vowel with given number. Use g to have the replace go through the string. 
```

Comment:
Would really like to see you use the terminology used for the expression (/a|A/g, 3) in your pseudocode.  While I love this approach, I want to make sure you understand all of the concepts used in this function.